scripts/algokit-libraries.sh

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env bash
+
+# SETUP
+PATH_TO_SOURCE_REPOS=$1 
+# ensure the following repos are checked out to the proper release/branch for doc generation and
+# ensure each is a subdirectory of $PATH_TO_SOURCE_REPOS 
+# git clone https://github.com/algorandfoundation/algokit-cli
+# git clone https://github.com/algorandfoundation/algokit-utils-ts
+# git clone https://github.com/algorandfoundation/algokit-utils-py
+# git clone https://github.com/algorandfoundation/algokit-client-generator-py
+# git clone https://github.com/algorandfoundation/algokit-client-generator-ts
+
+# REMOVE EXISTING ALGOKIT-LIBRARIES CONTENT
+PATH_TO_LIBRARIES=../docs/get-details/algokit/libraries
```

Comment:
If you will keep it as bash script, then i'd suggest to parametrize this argument as well